app/Migrations/Version20220726103404.php

```diff
@@ -22,7 +22,7 @@ public function up(Schema $schema): void
         $this->addSql('CREATE TEMPORARY TABLE __temp__sdk_setting AS SELECT id, path, "values", strategy, type, is_project, has_initialization, initialization_description, initializer FROM sdk_setting');
         $this->addSql('DROP TABLE sdk_setting');
         $this->addSql('CREATE TABLE sdk_setting (id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, path VARCHAR(255) NOT NULL, "values" CLOB DEFAULT NULL --(DC2Type:json)
-        , strategy VARCHAR(255) NOT NULL, type VARCHAR(255) NOT NULL, has_initialization BOOLEAN NOT NULL, initialization_description VARCHAR(255) DEFAULT NULL, initializer VARCHAR(255) DEFAULT NULL, setting_type CHECK(setting_type IN (\'sdk\',\'local\',\'shared\')))');
+        , strategy VARCHAR(255) NOT NULL, type VARCHAR(255) NOT NULL, has_initialization BOOLEAN NOT NULL, initialization_description VARCHAR(255) DEFAULT NULL, initializer VARCHAR(255) DEFAULT NULL, setting_type VARCHAR(10) CHECK(setting_type IN (\'sdk\',\'local\',\'shared\')))');
```

Comment:
We shouldn't fix migration in case it is released in that way, Instead, a new migration must be created